src/vcSettingsUI.cpp

```diff
@@ -474,7 +481,7 @@ void vcSettingsUI_Show(vcState *pProgramState)
           vcIGSW_FilePicker(pProgramState, vcString::Get("convertTempDirectory"), pProgramState->settings.convertdefaults.tempDirectory, udLengthOf(pProgramState->settings.convertdefaults.tempDirectory), nullptr, 0, vcFDT_SelectDirectory, [pProgramState] {
             // Nothing needs to happen here
             udUnused(pProgramState);
-          });
+            });
```

Comment:
You should change your VS settings for this, I think it's: `Options` -> `Text Editor` -> `C/C++` -> `Formatting` -> `Indentation` -> `In existing code, use the setting for alignment of new lines within parentheses`. Best way to test this is to reduce the indentation on this line and then apply formatting to the file - I'm fairly certain that triggers it, I remember it being kind of weird tracking down this setting.

Reply:
ta!